packages/valory/services/hello_world/service.yaml

```diff
@@ -0,0 +1,31 @@
+name: hello_world
+author: valory
+version: 0.1.0
+description: A simple demonstration of a simple ABCI application
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: QmPPeZtJ6aMXsGsCEynQcibxMRXypoe6K4KNAGiMi6JpaW
+fingerprint_ignore_patterns: []
+agent: valory/hello_world:0.1.0
+network: hardhat
```

Comment:
Can we specify network as none?

Reply:
Not atm. Might be worth adding support for

Reply:
#1001